packages/next/src/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -181,7 +181,7 @@ export class TraceEntryPointsPlugin implements webpack.WebpackPluginInstance {
       }
 
       // startTrace existed and callable
-      if (this.turbotrace) {
+      if (this.turbotrace && !this.turbotrace.skipEntries) {
```

Comment:
Won't this continue to leverage the non-turbotrace version instead of skipping entry tracing altogether? 

Also is this not necessary as no `fs` usage is present on the app and the `chunks` tracing is sufficient?

Reply:
> Won't this continue to leverage the non-turbotrace version instead of skipping entry tracing altogether?

Since turbotrace is about to go stable; do you think it's okay to only add it to turbotrace options?

> Also is this not necessary as no fs usage is present on the app and the chunks tracing is sufficient?

yes, so it's disabled by default.